src/test/fixtures/profiles/timings-with-js.js

```diff
@@ -86,4 +86,6 @@ const profile = {
   processes: [],
 };
 
-export default profile;
+export default function createProfile() {
+  return cloneDeep(profile);
+}
```

Comment:
here, instead of using cloneDeep, you can simply move lines 78 to 88 inside the function, because all properties are new. I see cloneDeep as a last resort mechanism :)